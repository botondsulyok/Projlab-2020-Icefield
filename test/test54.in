field spawn 0
entity esk Eskimo spawn
building tent1 Tent spawn
stats spawn
current
turn
stats spawn
current