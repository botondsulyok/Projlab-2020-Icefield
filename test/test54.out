id: spawn
type: iceunit
building: tent1
snow: 0
pickable: null
standers: esk
pullers: esk
masters: esk
neighbours: null

esk

id: spawn
type: iceunit
building: plains1
snow: 0
pickable: null
standers: esk
pullers: esk
masters: esk
neighbours: null

esk

